k8s/tests/scripts/run-tests.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+# Create namespace and setup RBAC
+kubectl create namespace "keramik-$1$TEST_SUFFIX" >/dev/null
+yq "
+ .metadata.name* = \"keramik-$1$TEST_SUFFIX\" |
+ (select(.subjects[] | length) | .subjects[0].name*) = \"keramik-$1$TEST_SUFFIX\" |
+ (select(.roleRef) | .roleRef.name) = \"keramik-$1$TEST_SUFFIX\"
+" ../manifests/setup.yaml | kubectl apply -f - >/dev/null
+
+# Create the network.
+kubectl create configmap check-network --from-file=check-network.sh -n "keramik-$1$TEST_SUFFIX" >/dev/null
+yq "
+  .metadata.name = \"$1$TEST_SUFFIX\"
+" ../../networks/"$1".yaml | kubectl apply -f - >/dev/null
+
+# Create the test job
+yq "
+  .spec.template.spec.serviceAccountName = \"keramik-$1$TEST_SUFFIX\"
+" ../manifests/prop.yaml | kubectl apply -n "keramik-$1$TEST_SUFFIX" -f - >/dev/null
+
+# Wait for tests to complete then collect the results
+kubectl wait --for=condition=complete job/tests -n "keramik-$1$TEST_SUFFIX" --timeout=20m >/dev/null
+TEST_RESULTS=$(kubectl logs job.batch/tests -c tests -n "keramik-$1$TEST_SUFFIX")
+echo "$TEST_RESULTS"
+# Return a failure from here instead of from the tests. The test job is setup to restart on failure, which is needed for
+# it to be setup correctly with the right Ceramic/ComposeDB URLs.
+if [[ "$TEST_RESULTS" =~ "FAILED" ]]; then
```

Comment:
This is fragile, I would prefer we use the exit code from the test itself. Maybe we can mess with the restart policy on the job? Or we could use a `kubectl wait` for the network itself so the test code doesn't have to do that.

Reply:
That's a great idea. Let me try that.